lib/src/chocolatey.dart

```diff
@@ -0,0 +1,279 @@
+// Copyright 2019 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'dart:io';
+
+import 'package:archive/archive.dart';
+import 'package:grinder/grinder.dart';
+import 'package:http/http.dart' as http;
+import 'package:path/path.dart' as p;
+import 'package:xml/xml.dart' as xml;
+import 'package:xml/xml.dart' hide parse;
```

Comment:
I wish this syntax were better. The intent is sort of reverse of what you have to write:

* import xml as xml (because I want `parse` as `xml.parse`)
* import xml hide parse (because I want all the things I don't enumerate on this line)

Unfortunately you don't want to enumerate all the classes you _do_ want, so the import which gives you `parse` doesn't mention it, and the import which gives you all the classes only mentions `parse`. :/

It'd be neat to have something like `import 'package:xml/xml.dart' classes;` or `import 'package:xml/xml.dart' functions as xml;`

Reply:
Agreed, although the library design is a big part of the problem—when you manually namespace *some* but not *all* of your members, you put downstream consumers in a tough spot no matter what. If they just renamed `parse()` to `parseXml()`, or removed the `Xml` prefix from all their classes so I could write `xml.Element`, this wouldn't be an issue.